example/app.js

```diff
@@ -13,14 +11,27 @@ APP = {
     }
 };
 
+jm = require('angular-jitsi-meet');
+
+angular
+    .module('app', [jm.name])
+    .controller('AppCtrl', AppCtrl);
+
 angular
-    .module('app', ['jm.xmpp'])
+    .module('app2', [jm.xmpp.name, jm.desktopsharing.name, jm.RTC.name, jm.Settings.name])
```

Comment:
the first `app` exists for bringing in all of the available modules and the second `app2` for bringing in a subset?  if so, it might be helpful to rename these to reflect this... something like `app_all` and `app_individual`.